src/main/groovy/pluggable/scm/bitbucket/BitbucketSCMProvider.groovy

```diff
@@ -147,7 +147,10 @@ public class BitbucketSCMProvider implements SCMProvider {
             break;
           }
 
-          String repoName = repo.substring(repo.lastIndexOf("/") + 1, repo.indexOf(".git"));
+          String repoName = repo.substring(repo.lastIndexOf("/") + 1, repo.length());
```

Comment:
I think we need to actually validate the repository URL (repo) here (e.g. is it a valid SSH/HTTPS clone URL).